<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Détails du Contact</string>
      </property>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="labelName">
         <property name="text">
          <string>Nom:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="lineEditName"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="labelPhone">
         <property name="text">
          <string>Téléphone:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="lineEditPhone"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="labelEmail">
         <property name="text">
          <string>Email:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="lineEditEmail"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="labelAddress">
         <property name="text">
          <string>Adresse:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="lineEditAddress"/>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QTableWidget" name="tableWidgetContacts">
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="buttonAdd">
        <property name="text">
         <string>Ajouter</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>:/icons/add.png</normaloff>
         </iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="buttonModify">
        <property name="text">
         <string>Modifier</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>:/icons/edit.png</normaloff>
         </iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="buttonDelete">
        <property name="text">
         <string>Supprimer</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>:/icons/delete.png</normaloff>
         </iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="buttonSearch">
        <property name="text">
         <string>Rechercher</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>:/icons/search.png</normaloff>
         </iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="buttonSort">
        <property name="text">
         <string>Trier</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>:/icons/sort.png</normaloff>
         </iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="buttonSave">
        <property name="text">
         <string>Sauvegarder</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>:/icons/save.png</normaloff>
         </iconset>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
</ui>
